pkgs/development/libraries/tbb/default.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
     sha256 = "1lygz07va6hsv2vlx9zwz5d2n81rxsdhmh0pqxgj8n1bvb1rp0qw";
   };
 
+  makeFlags = concatStringsSep " " (
+    optional (compiler != null) "compiler=${compiler}" ++
```

Comment:
How is that used by the build? If it's just for the binary $CC or cc can be used.

Reply:
This is used to select a makefile (which then determines the binary). It needs to have a specific name in ["gcc", "clang", "icc", "suncc", etc]. I'm not sure how to do this with $CC or cc as you suggested.

Reply:
was just wondering if we could get rid of the conditional, it seems to work fine on darwin at the moment tho (that uses clang by default)

Reply:
It works fine on darwin because the macos makefile sets the compiler to clang by default:
https://github.com/01org/tbb/blob/2018_U1/build/macos.inc#L60

Reply:
ah, nevermind the comment then.